assembl/views/api2/attachments.py

```diff
@@ -49,7 +49,7 @@ def get_file(request):
         .encode('iso-8859-1', 'replace'))
     url_quoted_utf8_filename = url_quote(f.title.encode('utf-8'))
     return Response(
-        body=f.data,
+        body=f.safe_data(),
```

Comment:
And if the resource is being called by multiple clients simultaneously when resource status is unchecked? There will be duplicate checks, and if antivirus machinery is unpredictable, the response of multiple checks could result in different status => headaches :(.

Reply:
Related to this, see my comment in safe_data(): https://github.com/assembl/assembl/pull/291/commits/901c07adef4b31aaeb0ea7979ae5dd5b472b85af#r159834353